### PR TITLE
Feature/listview template scrolltobottom

### PR DESCRIPTION
**Pull request by markrickert**
## Please verify the following:
- [x] Everything works on iOS/Android
- [x] `ignite-base` **ava** tests pass
- [x] `fireDrill.sh` passed

## Describe your PR

This adds the `scrollToBottom()` functionality to the `ListView` generator

### CODE REVIEW

File: ignite-generator/listview/templates/listview.js.template [hunk 1]
```diff
@@ -84,14 +89,50 @@ class <%= name %> extends React.Component {
...
+    }
+  }
+
+  // Save the list's height when it renders
+  onLayout = (event) => {
+    const layout = event.nativeEvent.layout
+    this.listHeight = layout.height
+  }
+
+  // Render a fake footer for determining the height of the ListView
```
Comment (GantMan) [inline ignite-generator/listview/templates/listview.js.template]: Can you change this to 'render a footer.  Keep `onFooterLayout` if you decide to fill out this section'

**Comment by GantMan** [comment]
small comment request made.  Plz add, and then 👍

**Comment by markrickert** [comment]
:shipit:

**Comment by GantMan** [comment]
We're going to merge this code, because we're still on 1.x which supports kitchen sink style.

Ideally, as our listview grows, we can move this off to our advanced generator.  Perhaps a question:  "Do you want helper functions?   scrollToBottom, pullToRefresh, etc."  once this grows in size.

Everyone please keep these ideas in mind, as we move forward 👍

**Pull request merged**